addons/l10n_ec_website_sale/views/website_sales_templates.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="partner_info" name="Ecuadorian partner">
+
+        <!-- show identification type -->
+        <div t-attf-class="mb-3 #{error.get('l10n_latam_identification_type_id') and 'o_has_error' or ''} col-xl-6">
+            <label class="col-form-label" for="l10n_latam_identification_type_id">Identification Type</label>
+            <t t-if="partner.can_edit_vat()">
+                <select name="l10n_latam_identification_type_id" t-attf-class="form-select #{error.get('l10n_latam_identification_type_id') and 'is-invalid' or ''}">
```

Comment:
It might be left as it is as it works althought not identically as the backend. In the backend the identification types are filtered depending on the selected country (the country is selected before the identification type), but in the portal the fields are ordered in a different manner.

Reply:
We can keep as it is right now, as it is done in other LATAM localizations.

Reply:
@vbe-odoo What happens if we install with another localization with identification types?

Reply:
Hey, @jco-odoo good point, I tested and it works well, mainly because the website is associated with the company and the country of the company. So if you access the eCommerce from the AR website, then you have the AR identification types, if you access the EC website then you have the EC ones.

So for me, this works OK.
video: https://www.loom.com/share/2c12122f1b1345ac827749bb8672dcaf?sid=ea52ec3a-c5b5-4b53-a7ed-974e0d15a508